### PR TITLE
add plugin issues to the scanner issues

### PR DESCRIPTION
**Pull request by mrmiywj**
Adding the issues found in the plugins besides the manifest.

### CODE REVIEW

**Comment by mrmiywj** [comment]
#279

File: qark/scanner/scanner.py [hunk 1]
```diff
@@ -68,14 +68,18 @@ def run(self):
...
+        plugins = (observer_plugin for observer_plugin in plugins if isinstance(observer_plugin, PluginObserver))
+        for plugin in plugins:
             current_file_subject.register(plugin)
 
         for filepath in self.files:
             current_file_subject.notify(filepath)  # All the plugins will be running now
             # reset the plugin file data to None
             current_file_subject.reset()
 
+        for plugin in plugins:
```
Comment (nwalsh-lnk) [inline qark/scanner/scanner.py]: I believe at this point the generator is going to be empty so that this won't do anything.

Comment (mrmiywj) [reply qark/scanner/scanner.py]: what do you mean generator mean?  In our test, this can help include in the issues found in other plugins.

Comment (mrmiywj) [reply qark/scanner/scanner.py]: Sorry, I did not notice that the `plugins` was a generator. I changed it to a list and ran a test. It works now.

**Comment by nwalsh-lnk** [comment]
Thanks for the PR!

**Pull request merged**